homeassistant/components/camera/onvif.py

```diff
@@ -183,3 +196,23 @@ def handle_async_mjpeg_stream(self, request):
     def name(self):
         """Return the name of this camera."""
         return self._name
+
+    @property
+    def device_state_attributes(self):
+        """Return the state attributes."""
+        attrs = super().device_state_attributes
+        custom_attrs = {}
+        i = 0
+        for profile in self._profiles:
```

Comment:
Of course that I can, but please think about it for one more time. User may not know what profiles camera supports and what is their video resolution. That's the case with my camera (Digoo) - I had no idea it supports more than one profile, until I started to analyse why I have artifacts, when trying to view the camera stream.
I thought that after looking at attributes user could find out, what are the profiles and which on suits best.
If you still think it is not necessary, let me know, I'll remove it.

Reply:
The problem is that it's static information. It's not part of the state. We're working on a different solution to store this info but it's not there yet.

Can't the user discover the profiles in the app ?

Reply:
In case of my camera - no. The official app displays information about 3 profiles (HD, SD, LQ). In fact there are only 2 profiles.
However there are unofficial apps that can do that.

If you're working on other solution to display device properties - I can remove profile info from attributes and add it later, when the feature is ready.